cluster-install/install.sh

```diff
@@ -1189,7 +1188,7 @@ create_kubectl_patch_for_containers() {
     [ -z "$resource_path" ] && fatal "no resource_path given"
     [ -z "$patch_value" ] && fatal "no patch_value given"
     if ! echo "$resource" | grep -q -- "/"; then fatal "invalid format for given resource '$resource': use 'kind/resource_name'"; fi
-    if ! echo "$patch_value" | jq &> /dev/null; then fatal "given patch_value '$patch_value' is not valid JSON"; fi
+    if ! echo "$patch_value" | jq '.' &> /dev/null; then fatal "given patch_value '$patch_value' is not valid JSON"; fi
```

Comment:
This is the `jq` change. In lower versions of JQ (observed at version 1.5.1 on Ajay's testing setup; my version is 1.7.1), a naked `jq` command simply prints the help/usage message, while later versions pretty-print (and as a side effect, validate) the piped input.